rivet-2.2.1.patch

```diff
@@ -0,0 +1,74 @@
+diff -Naur src/Analyses/ATLAS_2014_I1307243.cc srcNEW/Analyses/ATLAS_2014_I1307243.cc
+--- src/Analyses/ATLAS_2014_I1307243.cc	2015-03-19 14:25:42.000000000 +0100
++++ srcNEW/Analyses/ATLAS_2014_I1307243.cc	2015-04-27 11:13:43.321554321 +0200
+@@ -23,12 +23,16 @@
+       , _dy_max(8)
+       , _nEventsInAcceptance(0)
+       , _sumOfAcceptedWeights(0.)
++      , _fiducialRegions{2010, 2011}
++      , _vetoScale{20.*GeV, 30.*GeV}
++      , _yFiducial{4.4, 2.4}
++      , _gapCategories{"inclusive","gap"}
+     {
+       // Cannot do this in initialiser list without C++11
+-      _fiducialRegions = boost::assign::list_of<unsigned int>(2010)(2011);
+-      _vetoScale = boost::assign::list_of<double>(20.*GeV)(30.*GeV);
+-      _yFiducial = boost::assign::list_of<double>(4.4)(2.4);
+-      _gapCategories = boost::assign::list_of<std::string>("inclusive")("gap");
++//      _fiducialRegions = boost::assign::list_of<unsigned int>(2010)(2011);
```

Comment:
Could you please change the patch to just remove unused code instead of comment it?